core/api/src/services/ledger/schema.types.d.ts

```diff
@@ -19,10 +19,11 @@ interface ILedgerTransaction {
   txid?: string
   type: LedgerTransactionType
   pending: boolean
+  bundle_completion_state?: LnPaymentState
   err?: string
   currency: WalletCurrency
   feeKnownInAdvance?: boolean
-  related_journal?: ObjectId
```

Comment:
Changing this type could be a breaking change if there are already record in DB.

Reply:
We did find a few instances of ObjectId (likely from earlier code) in the db, so I added this mitigation to be safe https://github.com/GaloyMoney/galoy/pull/4039/commits/f558b2176189f894bbebcff9534240eb74969626